File: test/EntityFramework/VBTests/VbAdvancedPatterns/Partials/AdvancedPatternsModelFirstContext.vb
' Copyright (c) Microsoft Open Technologies, Inc. All rights reserved. See License.txt in the project root for license information.

Imports System.Data.Entity
Imports System.Data.Entity.TestHelpers

Namespace AdvancedPatternsVB

    <DbConfigurationType(GetType(FunctionalTestsConfiguration))> _
    Partial Friend Class AdvancedPatternsModelFirstContext

        Public Sub New(ByVal nameOrConnectionString As String)
            MyBase.New(nameOrConnectionString)
            MyBase.Configuration.LazyLoadingEnabled = False
        End Sub

    End Class

End Namespace

File: test/EntityFramework/FunctionalTests.Transitional/TestModels/TemplateModels/CsAdvancedPatterns/Partials/AdvancedPatternsModelFirstContext.cs
// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved. See License.txt in the project root for license information.

namespace FunctionalTests.ProductivityApi.TemplateModels.CsAdvancedPatterns
{
    internal partial class AdvancedPatternsModelFirstContext
    {
        public AdvancedPatternsModelFirstContext(string nameOrConnectionString)
            : base(nameOrConnectionString)
        {
            Configuration.LazyLoadingEnabled = false;
        }
    }
}
